tests/integration/REST/GetUsersWithPermissionInfoTest.php

```diff
@@ -0,0 +1,230 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Tests\Integration\AdminUi\REST;
+
+use Ibexa\Contracts\Core\Repository\RoleService;
+use Ibexa\Contracts\Core\Repository\UserService;
+use Ibexa\Contracts\Core\Repository\Values\User\UserCreateStruct;
+use Ibexa\Contracts\Test\Rest\WebTestCase;
+use LogicException;
+
+final class GetUsersWithPermissionInfoTest extends WebTestCase
+{
+    private const ENDPOINT_URL = 'permission/users-with-permission-info/%s/%s?%s';
+    private const HEADERS = [
+        'HTTP_ACCEPT' => 'application/json',
+        'X-Siteaccess' => 'admin',
+    ];
+    private const EDITOR_USER_GROUP_REMOTE_ID = '3c160cca19fb135f83bd02d911f04db2';
+    private const USER_REGISTRATION_REMOTE_ID = '5f7f0bdb3381d6a461d8c29ff53d908f';
+    private const MEDIA_CONTENT_ITEM_ID = 41;
+    private const MEDIA_LOCATION_ID = 43;
+    private const MODULE_CONTENT = 'content';
+    private const FUNCTION_EDIT = 'edit';
+    private const FUNCTION_READ = 'read';
+
+    private UserService $userService;
+
+    private RoleService $roleService;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $ibexaTestCore = $this->getIbexaTestCore();
+        $this->userService = $ibexaTestCore->getUserService();
```

Comment:
Shouldn't we rely on `IbexaKernelTestTrait::getServiceByClassName(YourClass::class)` instead?

Reply:
IMO since we introduced `ibexa/test-core` we should rely on newest test framework. `IbexaKernelTestTrait` comes from ibexa/core and AFAIK this is the old approach.